src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IepLevelResourceTest.kt

```diff
@@ -97,6 +97,14 @@ class IepLevelResourceTest : SqsIntegrationTestBase() {
       )
   }
 
+  @Test
+  fun `handle undefined path variable`() {
+    webTestClient.get().uri("/iep/reviews/booking/undefined")
```

Comment:
presumably as node passes `undefined` as a string due to dodgy concat, and this requires a long?

Reply:
Bingo!